blackbox/docs/sql/reference/deny.txt

```diff
@@ -0,0 +1,32 @@
+.. _ref-deny:
+
+========
+``DENY``
+========
+
+Denies privilege to an existing user on the whole cluster or on a specific object.
+
+Synopsis
+========
+
+.. code-block:: psql
+
+  DENY { { DQL | DML | DDL [,...] } | ALL [ PRIVILEGES ] }
+  [{ ON SCHEMA schema_name [, ...] |
+    ON TABLE  table_ident [, ...]  }]
+  TO user_name [, ...];
+
```

Comment:
i think this the synopsis needs to be changed in that that schema_name and table_ident are actually just identifiers. e.g:

... ON {TABLE | SCHEMA} identifier ....

additionally the cluster case needs to be documented here.

the same applies to REVOKE and GRANT